ribbon_test_environment/static/src/css/ribbon.css

```diff
@@ -0,0 +1,21 @@
+.test-ribbon{
+    width: 200px;
+    background: #e43;
+    position: absolute;
+    top: 25px;
+    left: -50px;
+    text-align: center;
+    line-height: 50px;
+    letter-spacing: 1px;
+    color: #f0f0f0;
+    transform: rotate(-45deg);
+    -webkit-transform: rotate(-45deg);
```

Comment:
Maybe you must include all the transform family prefixes (https://css-tricks.com/almanac/properties/t/transform/) to be sure the module works on all browsers,